modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/power_of_attorney_request_decisions_spec.rb

```diff
@@ -37,14 +38,27 @@
           ['Reason must be blank']
         )
       end
-    end
 
-    it 'creates acceptance decision with proper params' do
-      poa_request = create(:power_of_attorney_request)
+      it 'rep does not have poa for veteran' do
+        poa_request.update(power_of_attorney_holder_poa_code: '111',
+                           accredited_individual_registration_number: '1111111111')
 
-      post "/accredited_representative_portal/v0/power_of_attorney_requests/#{poa_request.id}/decision",
-           params: { decision: { type: 'acceptance', reason: nil } }
+        VCR.use_cassette('lighthouse/benefits_claims/power_of_attorney_decision/404_response.yml') do
```

Comment:
Possible that VCR comments on PR stack PR preceding this one will lead to changes here. I'll wait until later to review more closely.